help/destinations/catalog/advertising/magnite-batch.md

```diff
@@ -0,0 +1,190 @@
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+Read [Activate audience data to batch profile export destinations](/help/destinations/ui/activate-batch-profile-destinations.md) for instructions on activating audience segments to this destination.
+
+Once the destination connection/instance has been created, you will be shown the audience activation flow. The following process walks through how to activate audiences using the Magnite Streaming Batch destination:
+
+#### Step 1: Select your new destination connection/instance
+
+![select the destination to activate audiences too](../../assets/catalog/advertising/magnite/destination-batch-active-audience-select-destination.png)
+
+#### Step 2: Select any/all audiences you want to activate
+
+![select the audiences to activate](../../assets/catalog/advertising/magnite/destination-batch-active-audience-select-audience.png)
+
+select **[!UICONTROL Next]** to continue.
+
+### Map attributes and identities {#map}
+
+After selecting which audience(s) to activate, you will need to map your device attributes and/or identities. The following continues the steps above for how to map attributes/identities in the Magnite Streaming Batch destination:
+
+#### Step 3: Map your source device ID to a magnite streaming device ID
+
+In the "SOURCE FIELD", you can select any attribute or identity for your devices. In this example, we've selected a custom IdentityMap called "DeviceId"
+![map desired data fields to the device_id field](../../assets/catalog/advertising/magnite/destination-batch-active-audience-field-mapping.png)
+
+In the "TARGET FIELD":
+![select the appropriate device type target identity](../../assets/catalog/advertising/magnite/destination-batch-active-audience-select-device-type.png) See [Supported Identities](#supported-identities) for more information.
+In this example, we've selected the TARGET FIELD: magnite_deviceId_CUSTOM, because our SOURCE FIELD was definied as a custom IdentityMap: DeviceID. 
+
+**Note:** If you plan to send/map multiple ID types (GAID, IDFA, etc.) using the Batch destination, a new destination connection/instance is required for each. Please contact your Magnite Account representative for more information.
```

Comment:
Please use the correct note formatting: https://experienceleague.adobe.com/en/docs/contributor/contributor-guide/writing-essentials/markdown#note-blocks